docs/breadcrumbs-design.md

```
@@ -0,0 +1,257 @@
+Based on tenant feedback, apps would like to “inject” their breadcrumbs into the Chrome UI “breadcrumbs bar”. Because Chrome does not know what routes applications have and breadcrumbs are mostly dynamic, an API has to be created to allow apps to inject their partials to the root.
+
+# Current breadcrumbs
+
+Currently, chrome creates breadcrumbs only for routes that it “knows”. The known routes are determined from two config files:
+
+- fed-modules.json: global application registry available in an environment (stage, prod, beta, stable, EE)  
+- Navigation files for a bundle
+
+Chrome renders breadcrumbs to the last link available in the navigation files. Anything after that is a “black box” and Chrome can no longer guess the breadcrumbs titles or pathnames. It does not know what is eligible for routing and what is not.
+
+![][image1]
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)